Cognigy/chat-components
# Fit large images into chat bubble

This PR fixes an issue where a larger image would overflow over the chat bubble.

# How to Test

- `npm run dev`
- Open text section
- Check the last example
